go/cmd/aggregate-crls/aggregate-crls.go

```diff
@@ -116,6 +116,63 @@ func (ae *AggregateEngine) findCrlWorker(ctx context.Context, wg *sync.WaitGroup
 	resultChan <- issuerCrls
 }
 
+func (ae *AggregateEngine) crlFetchWorkerProcessOne(ctx context.Context, crlUrl url.URL, issuer storage.Issuer) (string, error) {
+	err := os.MkdirAll(filepath.Join(*crlpath, issuer.ID()), permModeDir)
+	if err != nil {
+		glog.Warningf("Couldn't make directory: %s", err)
+		return "", err
+	}
+
+	filename := makeFilenameFromUrl(crlUrl)
+	tmpPath := filepath.Join(*crlpath, issuer.ID(), filename+".tmp")
+	finalPath := filepath.Join(*crlpath, issuer.ID(), filename)
+
+	err = downloader.DownloadFileSync(ctx, ae.display, crlUrl, tmpPath, 3)
+	if err != nil {
+		glog.Warningf("[%s] Could not download %s to %s: %s", issuer.ID(), crlUrl.String(),
+			tmpPath, err)
+	} else {
+		// Validate the file and move it to the finalPath
+		cert, err := ae.issuers.GetCertificateForIssuer(issuer)
+		if err != nil {
+			glog.Fatalf("[%s] Could not find certificate for issuer: %s", issuer.ID(), err)
```

Comment:
Does `glog.Fatalf` terminate the program?

Reply:
It does, which will abort the pod and send alerts that the job couldn't complete.

This would be a pretty fatal error, indicating some failure in `issuers.go` to maintain state.